src/Microsoft.Crank.Controller/ScriptFile.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Crank.Controller
+{
+    public class ScriptFile
+    {
+        public string ReadFile(string filename)
+        {
+            if (String.IsNullOrEmpty(filename))
```

Comment:
```suggestion
            if (string.IsNullOrEmpty(filename))
```

https://blog.paranoidcoding.com/2019/04/08/string-vs-String-is-not-about-style.html

😉